etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -184,6 +184,15 @@ stackhpc_controller_firewalld_rules_template:
         network: "{{ octavia_net_name | default() }}"
         state: enabled
     enabled: "{{ kolla_enable_octavia | bool and octavia_net_name is not none }}"
+  # Mystery EIDF rules, required for creating VMs, need to check condition
```

Comment:
I don't think we should be naming customers within SKC.

As for these ports you can use

```
sudo lsof -i -P -n | grep LISTEN
```

To find what using these ports I assume either on the controllers/computes

Reply:
Port 67 is used by DHCP.

As for 6556 that port is used by [checkmk](https://checkmk.com/) which they do have a playbook for deploying.